subjects/printascii/README.md

```diff
@@ -0,0 +1,27 @@
+## print-ascii
+
+### instructions
+
+Write a program that prints the ASCII value of a character passed as an argument to the program
+- If the argument is not a letter nothing will be printed
```

Comment:
character word is a broad word, it can represent text, numbers, or symbols as long as they are stored in one byte, maybe replace it with a printable character and accept all the range 32<= x < 127 or to an English alphabet letter

Reply:
in this exercise, the student should print ASCII of letters only I write it : 
> If the argument is not a letter nothing will be printed